### PR TITLE
Verifies that SDL does not put RC apps to NONE state when user disables RC in HMI

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Test for verifying of the issue https://github.com/smartdevicelink/sdl_core/issues/2173

### CODE REVIEW

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua [hunk 1]
```diff
@@ -12,9 +12,8 @@
 --
 -- SDL must:
 -- 1) store RC state allowed:false internally
--- 2) assign HMILevel none to all registered applications with appHMIType REMOTE_CONTROL and send OnHMIStatus (NONE) to such apps
--- 3) keep all applications with appHMIType REMOTE_CONTROL registered
--- 4) unsubscribe all REMOTE_CONTROL applications from OnInteriorVehicleData notifications for all HMI modules internally
+-- 2) keep all applications with appHMIType REMOTE_CONTROL registered
```
Comment (GetmanetsIrina) [inline test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua]: Propose to update to "keep all applications with appHMIType REMOTE_CONTROL registered and in current HMI levels "

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua [hunk 2]
```diff
@@ -50,8 +49,8 @@ local function disableRcFromHmi(self)
 
 	commonRC.defineRAMode(false, nil, self)
 
-	mobileSession1:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE"}):Times(AtLeast(1)) -- issue with SDL --> notification is sent twice
-	mobileSession2:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE"}):Times(AtLeast(1)) -- issue with SDL --> notification is sent twice
+	mobileSession1:ExpectNotification("OnHMIStatus"):Times(0)
+	mobileSession2:ExpectNotification("OnHMIStatus"):Times(0)
```
Comment (GetmanetsIrina) [inline test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua]: Please use alignment the same through all script.

**Comment by LitvinenkoIra** [comment]
@GetmanetsIrina please look at 4d511b641d0af943ceb47664eb3b130af84249ec

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/024_Allowed_false_no_PTU.lua [hunk 3]
```diff
@@ -13,8 +13,7 @@
 -- SDL must:
```
Comment (dboltovskyi) [inline test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/024_Allowed_false_no_PTU.lua]: @LitvinenkoIra Please add link to updated req-t (as in line 4 in script 008)

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua [hunk 4]
```diff
@@ -77,11 +76,11 @@ for _, mod in pairs(modules) do
   -- Apps are not subscribed from RC modules
   runner.Step("Check App1 is not subscribed on " .. mod, commonRC.isUnsubscribed, { mod, 1 })
   runner.Step("Check App2 is not subscribed on " .. mod, commonRC.isUnsubscribed, { mod, 2 })
-  -- -- All RC RPCs denied - need clarification
-  -- for _, rpc in pairs(rcRpcs) do
-  --   runner.Step("Check module " .. mod .." App1 " .. rpc .. " denied", commonRC.rpcDenied, { mod, 1, rpc, "DISALLOWED" })
-  --   runner.Step("Check module " .. mod .." App2 " .. rpc .. " denied", commonRC.rpcDenied, { mod, 2, rpc, "DISALLOWED" })
-  -- end
+  -- All RC RPCs denied
```
Comment (dboltovskyi) [inline test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua]: @LitvinenkoIra Please update that RPCs are not denied, but rejected

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/024_Allowed_false_no_PTU.lua (hunk 3 above)
Comment (LitvinenkoIra) [reply test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/024_Allowed_false_no_PTU.lua]: @dboltovskyi Done db789fa84af61fc45231bb4dbae583598fa4f55c

File: test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua (hunk 4 above)
Comment (LitvinenkoIra) [reply test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/008_Allowed_false.lua]: @dboltovskyi Done db789fa84af61fc45231bb4dbae583598fa4f55c

**Pull request merged**